catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/backpack/BackpackRecyclerViewFragment.java

```diff
@@ -292,28 +298,40 @@ public void onItemClick(final T item, MultiSelectionManager selectionManager) {
 			return;
 		}
 
-		List<Integer> options = new ArrayList<>();
-		options.add(R.string.unpack);
-		options.add(R.string.delete);
-		List<String> names = new ArrayList<>();
-		for (Integer option: options) {
-			names.add(getString(option));
-		}
-		ListAdapter arrayAdapter = UiUtils.getAlertDialogAdapterForMenuIcons(options,
-				names, requireContext(), requireActivity());
-
-		new AlertDialog.Builder(requireContext())
-				.setTitle(getItemName(item))
-				.setAdapter(arrayAdapter, (dialog, which) -> {
-					switch (which) {
-						case 0:
-							unpackItems(new ArrayList<>(Collections.singletonList(item)));
-							break;
-						case 1:
-							showDeleteAlert(new ArrayList<>(Collections.singletonList(item)));
-					}
-				})
-				.show();
+		Pair[] items = new Pair[] {
+				new Pair<String, Integer>(getString(R.string.unpack), R.drawable.ic_logout),
+				new Pair<String, Integer>(getString(R.string.delete), R.drawable.ic_delete)};
```

Comment:
It would be nice to get rid of the warning `Raw use of parameterized class 'Pair'` by using Lists as bellow:
```suggestion
		List<Pair<String, Integer>> items = Arrays.asList(
				new Pair<>(getString(R.string.unpack), R.drawable.ic_logout),
				new Pair<>(getString(R.string.delete), R.drawable.ic_delete)
		);
```